minimum-constraints.txt

```diff
@@ -44,7 +44,7 @@ wheel==0.29.0
 
 # Direct dependencies for runtime (must be consistent with requirements.txt)
 
-pywbem==0.12.6
+pywbem==0.13.0
```

Comment:
This version 0.13.0 here is inconsistent with version 0.14.0 in requirements.txt.

Reply:
This is actually a comment on the other PR.